release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,93 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-server-config {
+    description
+      "Configuration data for the netconf server/subsystem";
+
+    leaf netconf-transport-porotocol {
```

Comment:
mispelled `protocol` and why would this be prefixed with `netconf-` ? This does not need to be reiterated

Reply:
Corrected the spelling and leaf name